subsys/logging/log_output_syst.c

```diff
@@ -24,8 +24,8 @@
 
```

Comment:
you probably want to fold this commit into the previous commit introducing this file.